pallets/subtensor/src/macros/errors.rs

```diff
@@ -210,6 +210,9 @@ mod errors {
         InvalidRecoveredPublicKey,
         /// SubToken disabled now
         SubtokenDisabled,
+        /// Too frequent hotkey swap on subnet
+        HotKeySwapOnSubnetIntervalNotPassed,
+        /// Estimating the maximum stake for limited staking operations returned zero.
         /// Zero max stake amount
```

Comment:
There is an issue with the comment here, no? Should be line 215 only
```suggestion
        /// Estimating the maximum stake for limited staking operations returned zero.
```

Reply:
should be a mistake, will add it back.